benchmark/Streamly/Benchmark/Prelude/Adaptive.hs

```diff
@@ -81,51 +81,51 @@ main =
   defaultMain
     [
       bgroup "serialConstantSlowConsumer"
-      [ bench "serially"    $ nfIO $ alwaysConstSlowSerial serially
-      , bench "wSerially"   $ nfIO $ alwaysConstSlowSerial wSerially
+      [ bench "fromSerial"    $ nfIO $ alwaysConstSlowSerial fromSerial
```

Comment:
In the benchmark names it probably makes better sense to call it "serial" or "serially" rather than "fromSerial". We are basically saying that we are benchmarking a "serial" stream.